lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -32,116 +29,43 @@ class LoadFastEnough4Pwa extends Audit {
-        const fromCache = record._fromDiskCache || record._fromMemoryCache;
-        const origin = URL.getOrigin(record._url);
-        if (!origin || !record._timing || fromCache ||
-            WHITELISTED_STATUS_CODES.includes(record.statusCode) || !record.finished) {
-          return;
-        }
-
-        // Disregard requests with an invalid start time, (H2 request start times are sometimes less
-        // than issue time and even negative which throws off timing)
-        if (record._startTime < record._issueTime) {
-          return;
-        }
-
-        // Use DevTools' definition of Waiting latency: https://github.com/ChromeDevTools/devtools-frontend/blob/66595b8a73a9c873ea7714205b828866630e9e82/front_end/network/RequestTimingView.js#L164
-        const latency = record._timing.receiveHeadersEnd - record._timing.sendEnd;
-        const latencyInfo = {
-          url: record._url,
-          startTime: record._startTime,
-          origin,
-          latency,
-        };
-
-        // Only examine the first request per origin to reduce noisiness from cases like H2 push
-        // where individual request latency may not apply.
-        const existing = firstRequestLatenciesByOrigin.get(origin);
-        if (!existing || latencyInfo.startTime < existing.startTime) {
-          firstRequestLatenciesByOrigin.set(origin, latencyInfo);
-        }
-      });
-
-      let firstRequestLatencies = Array.from(firstRequestLatenciesByOrigin.values());
-      const latency3gMin = targetLatencyMs - 10;
-      const areLatenciesAll3G = firstRequestLatencies.every(val => val.latency > latency3gMin);
-      firstRequestLatencies = firstRequestLatencies.map(item => ({
-        url: item.url,
-        latency: Util.formatNumber(item.latency, 0.01),
-      }));
-
-      const trace = artifacts.traces[Audit.DEFAULT_PASS];
-      return artifacts.requestFirstInteractive(trace).then(firstInteractive => {
-        const timeToFirstInteractive = firstInteractive.timeInMs;
-        const isFast = timeToFirstInteractive < MAXIMUM_TTFI;
-
-        const extendedInfo = {
-          value: {areLatenciesAll3G, firstRequestLatencies, isFast, timeToFirstInteractive},
-        };
+  static async audit(artifacts, context) {
+    const trace = artifacts.traces[Audit.DEFAULT_PASS];
+    const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    const settingOverrides = {throttlingMethod: 'simulate', throttling: mobile3GThrottling};
+    const settings =
+      context.settings.throttlingMethod !== 'provided' &&
+      isDeepEqual(context.settings.throttling, mobile3GThrottling)
+        ? context.settings
+        : Object.assign({}, context.settings, settingOverrides);
+    const metricComputationData = {trace, devtoolsLog, settings};
+    const tti = await artifacts.requestConsistentlyInteractive(metricComputationData);
+
+    const score = Number(tti.timing <= MAXIMUM_TTI);
```

Comment:
https://developers.google.com/web/progressive-web-apps/checklist does say 

> interactive <10s for first visit on a simulated 3G network.

let's change this to `<` just to simplify. nice to stay single digits. :)

Reply:
sg :)